src/classes/STG_InstallScript.cls

```diff
@@ -112,6 +115,8 @@ global without sharing class STG_InstallScript implements InstallHandler {
             } else if (action == InstallAction.CREATE_DEFAULT_SETTINGS) {
                 createNewDefaultCustomSettingRecords();
 
+            } else if (action == InstallAction.POPULATE_NAMESPACE_PREFIX) {
```

Comment:
Though, it may be that we need to execute this logic on both a push and a brand new install. For example, if someone were to install Cumulus package directly into their org (not as part of a trial), then this object would be empty and the buttons would not work. So, maybe the additional fix here is a new Enum for `NEW_INSTALL_CUSTOM_SETTINGS` that calls your method. That action would get included in the `context.previousVersion() == null` part of the `onInstall` method.

Reply:
We need it both for new installs and for push upgrades (at least the first push upgrade after we add it to the package, but it shouldn't hurt to do it each time). Are you saying call the method both from `NEW_INSTALL_CUSTOM_SETTINGS` and from `CREATE_DEFAULT_SETTINGS`?

Reply:
Yes. I think that will work. For clarity, you could rename `CREATE_DEFAULT_SETTINGS` to `CREATE_DEFAULT_SETTINGS_IN_UPGRADE`.